tests/text/test_tabstops.py

```diff
@@ -34,9 +34,30 @@ def it_can_iterate_over_its_tab_stops(self, iter_fixture):
         assert count == expected_count
         assert TabStop_.call_args_list == expected_calls
 
+    def it_can_get_a_tab_stop_by_index(self, index_fixture):
+        tab_stops, expected_vals, tab_stop_, TabStop_, expected_calls = (
+            index_fixture
+        )
+        for count in range(len(tab_stops)):
+            assert tab_stops[count] is tab_stop_
+        assert TabStop_.call_args_list == expected_calls
+
```

Comment:
> In tests/text/test_tabstops.py 
> <https://github.com/python-openxml/python-docx/pull/261#discussion_r61119742>:
>
> > @@ -34,9 +34,30 @@ def it_can_iterate_over_its_tab_stops(self, iter_fixture):
> >          assert count == expected_count
> >          assert TabStop_.call_args_list == expected_calls
> >
> > +    def it_can_get_a_tab_stop_by_index(self, index_fixture):
> > +        tab_stops, expected_vals, tab_stop_, TabStop_, expected_calls = (
> > +            index_fixture
> > +        )
> > +        for count in range(len(tab_stops)):
> > +            assert tab_stops[count] is tab_stop_
> > +        assert TabStop_.call_args_list == expected_calls
> > +
>
> Did you replace this test method with the version I provided above? 
> There's only one call (no for loop anymore), so I can't see how it 
> would get called twice.
>

Yes, I replaced it.  But I don't know how to write the fixture.  For 
example, the fixture returns tab.  What's the value that's supposed to 
be in that parameter?  I don't know what is supposed to be passed to 
TabStop_.assert_called_once_with(tab) and I've got no clue how to figure 
it out.

I've spent all morning reading about Mock, but nothing I've found to 
read has helped even slightly.  I can't find a way in.

David

> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/python-openxml/python-docx/pull/261/files/028517fe35492cef4ea19e0e58301ee2e4d14cc5..5668a5dd1c7feda9b5c3f1859752318cc9baf8f9#r61119742>
>